scout/server/blueprints/variants/templates/variants/utils.html

```diff
@@ -210,15 +210,17 @@
       {{ form.genetic_models(class="selectpicker", data_style="btn-secondary") }}
     </div>
   </div>
-  <div class="row mt-2">
+  <div class="row mb-2">
     <div class="col-3">
       {{ form.hgnc_symbols.label(class="control-label") }}
       {{ form.hgnc_symbols(class="form-control") }}
     </div>
-    <div class="col-1 d-flex align-items-end">
-        <a class="btn btn-secondary" href="{{ url_for('variants.variants', institute_id=institute._id,
+    <div class="col-1 align-self-end">
+      <div class="d-flex btn-group">
```

Comment:
So it's the same as the button in the SNV variants form I guess?
OK! 👍🏻

Reply:
Exactly.. The alignment to the bottom line was a little off, but its mostly the reactive nature of it, and not having fixed screen sizes. When other buttons further down the line get squeezed, the end moves down a little more.